assets/jwt-ptf.yaml

```diff
@@ -38,10 +37,12 @@ spec:
         image: 'cyberark/secrets-provider-for-k8s:latest'
         imagePullPolicy: Always
         volumeMounts:
+        - name: conjur-status
+          mountPath: /conjur/status
         - name: podinfo
           mountPath: /conjur/podinfo
         - name: conjur-secrets
-          mountPath: /conjur/secrets
+          mountPath: /conjur/
```

Comment:
Will mounting the secrets volume at `/conjur` overwrite the other volume mounts at `/conjur/status` and other subdirectories?

Reply:
Yea they are using /conjur and not what we were recommending. The docs are being re designed so we won't have so many of these manifests. The final one should have /conjur/secrets. Will leave this as is for now as a copy.